src/content/docs/fr/guides/backend/sentry.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Surveillez votre site Astro avec Sentry
+description: Comment surveiller votre site Astro avec Sentry
+type: backend
+service: Sentry
+stub: false
+i18nReady: true
+---
+import ReadMore from '~/components/ReadMore.astro';
+import { Steps } from '@astrojs/starlight/components';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Sentry](https://sentry.io) offre un service complet de surveillance des applications et de suivi des erreurs, conçu pour aider les développeurs à identifier, diagnostiquer et résoudre les problèmes en temps réel.
+
+Pour en savoir plus, consultez notre blog sur [le partenariat entre Astro et Sentry](https://astro.build/blog/sentry-official-monitoring-partner/) et la barre d'outils Spotlight de Sentry, qui apporte une riche couche de débogage dans votre environnement de développement Astro. Spotlight affiche les erreurs, les traces et le contexte important directement dans votre navigateur pendant le développement local.
+
+Le SDK Astro de Sentry permet de signaler automatiquement les erreurs et les données de traçage dans votre application Astro.
+
+## Configuration du projet
+
+Une liste complète des prérequis est disponible dans [le guide Sentry pour Astro](https://docs.sentry.io/platforms/javascript/guides/astro/#prerequisites).
+
+## Installation
+
+Sentry capture des données en utilisant un SDK dans le runtime de votre application.
+
+Installez le SDK en exécutant la commande suivante pour le gestionnaire de paquets de votre choix dans le CLI Astro :
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+    ```shell
+    npx astro add @sentry/astro
+    ```
+  </Fragment>
+  <Fragment slot="pnpm">
+    ```shell
+    pnpm astro add @sentry/astro
+    ```
+  </Fragment>
+  <Fragment slot="yarn">
+    ```shell
+    yarn astro add @sentry/astro
+    ```
+  </Fragment>
+</PackageManagerTabs>
+
+L'astro CLI installe le paquet SDK et ajoute l'intégration Sentry à votre fichier `astro.config.mjs`.
```

Comment:
I think "astro CLI" should be replaced with "Le CLI d'Astro" to be consistent with the [following page title](https://docs.astro.build/fr/reference/cli-reference/) in the nav.